src/dotnet/Chat.Service/ChatsBackend.cs

```diff
@@ -490,6 +509,43 @@ public virtual async Task OnNewUserEvent(NewUserEvent @event, CancellationToken
         return attachment;
     }
 
+    // [CommandHandler]
+    public virtual async Task React(IChatsBackend.ReactCommand command, CancellationToken cancellationToken)
```

Comment:
@FrolYo if you want to do it this way, IMO you need at least `HasReactions` flag in `DbChatEntry` - otherwise it's going to be super expensive to fetch these.

Reply:
And eventually we'll also need a real summary (i.e. with counts) stored in the DB. But it's fine for now, I guess.

Reply:
done